src/routes/+page.svelte

```diff
@@ -1,59 +1,51 @@
 <script>
-	import Counter from './Counter.svelte';
-	import welcome from '$lib/images/svelte-welcome.webp';
-	import welcome_fallback from '$lib/images/svelte-welcome.png';
-</script>
-
-<svelte:head>
-	<title>Home</title>
-	<meta name="description" content="Svelte demo app" />
-</svelte:head>
 
-<section>
-	<h1>
-		<span class="welcome">
-			<picture>
-				<source srcset={welcome} type="image/webp" />
-				<img src={welcome_fallback} alt="Welcome" />
-			</picture>
-		</span>
+let seed = '';
+
+// Generates a 7 random letter seed from a-z and A-Z
+const generateSeed = () => {
+    const allLetters = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ';
+    for(let i = 0; i < 7; i++) {
+        seed += allLetters.charAt(Math.floor(Math.random() * allLetters.length));
+    }
+    generateRandomValues(seed);
```

Comment:
Its fine to leave this function call in for testing, but it can be removed later, since we will just call it where and when we need it.